translations/ui-marc-authorities/en.json

```diff
@@ -10,6 +10,9 @@
   "actions.menuSection.sortBy.relevance": "Relevance",
 
   "authority-record.edit": "Edit",
+  "authority-record.delete": "Delete",
+  "authority-record.delete.success": "MARC authority record {id} has been deleted",
```

Comment:
As in my other comment - we should place all markup inside translations and not pass it to `<FormattedMessage>`
So the translations should look like:
`authority-record.delete.success": "MARC authority record <strong>{id}</strong> has been deleted"`